README.md

```diff
@@ -14,26 +14,9 @@ Subscribe to the following repo to be notified of announcements and updates abou
 
 ## REST API Reference
 
-The following reference pages are available to describe the Azure App Configuration API surface in detail.
-
-**Resources**
-  * [Keys](./docs/REST/keys.md)
-  * [Key-Values](./docs/REST/kv.md)
-  * [Labels](./docs/REST/labels.md)
-  * [Locks](./docs/REST/locks.md)
-  * [Revisions](./docs/REST/revisions.md)
-
-**Protocol**
-  * [Authentication](./docs/REST/authentication/index.md)
-  * [Authorization](./docs/REST/authorization/index.md)
-  * [Consistency Model](./docs/REST/consistency.md)
-  * [Common Headers](./docs/REST/headers.md)
-  * [Throttling](./docs/REST/throttling.md)
-  * [Versioning](./docs/REST/versioning.md)
-
-**Development**
-  * [Fiddler](./docs/Development/fiddler.md)
-  * [Postman](./docs/Development/postman.md)
+Follow the links below for the Azure App Configuration REST API reference.
+  * [Key-vaule data operation](https://docs.microsoft.com/azure/azure-app-configuration/rest-api)
```

Comment:
vaule -> value

Reply:
Thanks for catching it, @weshaggard :)

Reply:
I only happened upon this because this PR broken some links in some of our app config samples. I just fixed up https://github.com/Azure/azure-sdk-for-net/pull/17127.

Reply:
Yes, pages on Github tend to change more often. With REST API references moved to the official docs site, we also hope those links can be more 'permanent' and provide better discovery.